front/pages/api/auth/[auth0].ts

```diff
@@ -6,24 +6,24 @@ import {
   handleLogout,
   IdentityProviderError,
 } from "@auth0/nextjs-auth0";
-import type { AuthorizationParameters } from "@auth0/nextjs-auth0/dist/auth0-session";
 import type { NextApiRequest, NextApiResponse } from "next";
 
 import config from "@app/lib/api/config";
 
 export default handleAuth({
   login: handleLogin((req) => {
-    const connection = "query" in req ? req.query.connection : undefined;
+    let connection: string | undefined = undefined;
+    if ("query" in req && req.query.connection) {
+      connection = Array.isArray(req.query.connection)
```

Comment:
When is this happening?

Reply:
req.query can be a string as per typescript you can do req[]=1&req[]=2 I believe

Reply:
I see, common pattern we use in the code is if `!== string` we return an error. In this case you can redirect to `      return res.redirect("/login-error");`.